core/vm/runtime/evm_code_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2015 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package runtime
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+var (
+	codefile string
+	input    string
+	expected string
+)
+
+var _ = func() bool {
+	testing.Init()
+	flag.StringVar(&codefile, "codefile", "", "EVM code to run")
+	flag.StringVar(&input, "input", "", "input calldata")
+	flag.StringVar(&expected, "expected", "", "expected return data")
+	flag.Parse()
+	return true
+}()
+
+func BenchmarkEvmCode(b *testing.B) {
+
+	state, _ := state.New(common.Hash{}, state.NewDatabase(rawdb.NewMemoryDatabase()))
+	address := common.HexToAddress("0x0a")
+
+	var (
+		codebytes []byte
+		err       error
+		ret       []byte
+		codehex   []byte
+		gasLeft   uint64
+	)
+
+	fmt.Println("codefile:", codefile)
```

Comment:
Do you really need all these `fmt.Println` ? Why not just show these in case there's a failure?